settings.gradle.kts

```diff
@@ -17,10 +17,24 @@ pluginManagement {
     }
 }
 
-include(
-        ":samples:junit-tests",
+include(":samples:junit-tests")
+
+if (isAndroidSDKConfigured()) {
+    include(
         ":samples:kotlin-android",
         ":samples:kotlin-coroutines",
-        ":samples:groovy-android")
+        ":samples:groovy-android"
+    )
+}
 
 includeBuild("gradle-plugin")
+
+fun isAndroidSDKConfigured(): Boolean {
+    if (System.getenv("ANDROID_SDK_ROOT") != null) {
```

Comment:
```suggestion
    if (System.getenv("ANDROID_SDK_ROOT") != null || System.getenv("ANDROID_HOME") != null) {
```

Reply:
I'm surprised about `ANDROID_HOME`. When running it (on a machine without android) I get an error message referring to `ANDROID_SDK_ROOT` but with NO mentions to `ANDROID_HOME`

Reply:
From: https://developer.android.com/studio/command-line/variables

```
ANDROID_SDK_ROOT | Sets the path to the SDK installation directory. Once set, the value does not typically change, and can be shared by multiple users on the same machine. ANDROID_HOME, which also points to the SDK installation directory, is deprecated.
```

So yeah let's take both just for backward compat.
Also the GH runner has both set correctly.